src/infohud.asm

```diff
@@ -323,10 +323,10 @@ ih_update_hud_code:
         }
 
         ; Draw seconds
-        LDA !ram_tmp_1 : JSR Hex2Dec : LDX #$003C : JSR Draw3
+        LDA !ram_tmp_1 : LDX #$003C : JSR Draw3
```

Comment:
I combined Hex2Dec with the Draw methods, which made the methods more efficient.  It also allows for better number overflow.  If a two-digit or three-digit number overflows, it will use the new 10-65 characters.